src/http/request.ts

```diff
@@ -56,7 +53,7 @@ export class DrashRequest extends Request {
   }
 
   get original(): Request {
-    return this.#original;
+    return Object.getPrototypeOf(this);
```

Comment:
Wait... I'm kind of lost. We can't clone the original request? Doesn't `Object.getPrototypeOf(this)` return the current instance of the request? So, if a request is changed, the prototype will also be changed, right?

Reply:
we changed the approach in #691 @Guergeiro. i just merged #691 into this PR.